packages/web/package.json

```diff
@@ -10,6 +10,7 @@
     "lint:styles": "stylelint 'src/**/*.@(ts|tsx|js)'",
     "lint:types": "tsc --noEmit",
     "precommit": "lint-staged",
+    "prepare:contracts": "cp -r ../contracts/deployments ./contracts",
```

Comment:
it's supposed to be manual for now right? because is not called anywhere

Reply:
Yes, we will deal with that later.